src/stylesheets/base/_typography.scss

```diff
@@ -0,0 +1,19 @@
+@font-face {
+    font-family: geomanist;
+    src: url(fonts/geomanist-regular-webfont.woff2);
+}
+
+@font-face {
+    font-family: geomanist-medium;
+    src: url(fonts/geomanist-medium-webfont.woff2);
+}
```

Comment:
This is personal preference but I like to set up all my variables (colors, fonts, sizes, etc) in a separate file. Up to you how you organize things though